sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/client/TokenExchange.java

```diff
@@ -110,7 +111,14 @@ public void onBeforeCalling(URL url, Map<String, String> headers) {
         @Override
         public void onCallSucceeded(String payload, Map<String, String> headers) {
             final TokenResult tokenResult = parseTokenResult(payload);
```

Comment:
We usually don't use final at local or parameter level if we don't have to.
We use it if we can at member field or static level.